src/vorta/borg/_compatibility.py

```diff
@@ -1,7 +1,8 @@
 from pkg_resources import parse_version
 
 MIN_BORG_FOR_FEATURE = {
-    'REPOKEY_BLAKE2': parse_version('1.1.4'),
+    'BLAKE2': parse_version('1.1.4'),
```

Comment:
1.1.0

Reply:
Didn't you say there were some issues with blake2 before 1.1.4? So I figured I use the version where it was rock-solid.